doc/cards.xsd

```diff
@@ -29,12 +29,12 @@
                           <xs:extension base="xs:string">
                             <xs:attribute type="xs:int" name="muId" use="optional"/>
                             <xs:attribute type="xs:anyURI" name="picUrl" use="optional"/>
-                            <xs:attribute type="xs:anyURI" name="picUrlHq" use="optional"/>
```

Comment:
Changes included in this PR removes che choice between SQ and HQ images; you can get them form the "default source" (currently, the gatherer) or a custom source.